### PR TITLE
lab-cross-validation Jack

### PR DESCRIPTION
**Pull request by jack93g**

### CODE REVIEW

**Comment by AnaAGG** [comment]
# lab cross validation

Helloooo Jack 🙋🏻‍♀️🤓! Let's go with the cross validation lab

- One thing that I have missed is the transformation of the predictor variables. As we are working with continuous variables we need to check whether there is a need to normalise or standardise predictor variables. 

- On the other hand, if you see you repeaeted the same code all the time, only changing the data. Why can we do?

    Create functions!!! 

    So.. in this case you could code:

    ```python
    def classification_models(X_train, y_train, X_test, y_test, model):
        model.fit(X_train, y_train)
        
        score_class = model.score(X_test, y_test)
        
        predictions = model.predict(X_test)
        value_cnt_class = pd.Series(predictions).value_counts()
        
        print('The accuracyscore is: ', score_class, '\n')
        print('Value counts of the model predictions: ', '\n', value_cnt_class)

    # and then call the function in this way: 
    classification_models(X_train_smote_lr, y_train_smote_lr, X_test, y_test, LogisticRegression(random_state=12, max_iter=100))
    ```

Come on Jack!!! We'll keep going, there's nothing left 💪!

image
